src/benchmarks/real-world/ImageSharp/TestUtilities/TestEnvironment.cs

```diff
@@ -0,0 +1,282 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace SixLabors.ImageSharp.Tests
+{
+    public static partial class TestEnvironment
```

Comment:
You may want to drop the unused bits from here. (The original file is our test infra god-object.)